modules/repository-url/src/test/resources/org/opensearch/bootstrap/test.policy

```diff
@@ -0,0 +1,11 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+grant {
```

Comment:
@andrross why do you need this policy?

Reply:
@reta I honestly don't know why it is needed now. The [URLBlobStoreTests](https://github.com/opensearch-project/OpenSearch/blob/34c860d82be75136ce3cef60fcd9523f1e86afd5/modules/repository-url/src/test/java/org/opensearch/common/blobstore/url/URLBlobStoreTests.java#L70) start an http server on the loopback address and [this call](https://github.com/opensearch-project/OpenSearch/blob/411b279e8d8c25ff26a961faacf0ee1fd9d0e612/modules/repository-url/src/main/java/org/opensearch/common/blobstore/url/URLBlobContainer.java#L164) fails without this policy.

Reply:
Actually, I think it might be because we're not truncating the stack like we were with the security manager. It seems like the junit jar needs this permission where previously it did not.

Reply:
It already is in [plugin-security.policy](https://github.com/opensearch-project/OpenSearch/blob/b15ffc9caaec0303b91ecaff1ab39bb9e860ba48/plugins/discovery-ec2/src/main/plugin-metadata/plugin-security.policy#L42). I think the change here is that the test runner (junit) itself needs the permission where previously it did not.

The same change is needed in repository-s3 and discovery-ec2.